### PR TITLE
Feat/login + edit redux action structure

### PR DESCRIPTION
**Pull request by bcuz**
- Wilfred is planning on making a new register component, so I recreated the login component. The previous login component included registration and login together
- basic login with 3 actions
- separate files for actions
- auth0 stuff in next PR

### CODE REVIEW

File: socialtours/src/actions/index.js [hunk 1]
```diff
@@ -26,7 +26,11 @@ export const types = {
 	// Completed by Greg,
 	DELETE_EVENT: "DELETE_EVENT",
 	DELETE_EVENT_SUCCESS: "DELETE_EVENT_SUCCESS",
-	DELETE_EVENT_FAILED: "DELETE_EVENT_FAILED"
+	DELETE_EVENT_FAILED: "DELETE_EVENT_FAILED",
```
Comment (RaneWallin) [inline socialtours/src/actions/index.js]: I don't think all of our actions should be in one file. I think there should be a login actions file and also probably an events actions file, then we can pull all of that into the index if needed.

**Comment by bcuz** [comment]
going to separate the actions

**Comment by bcuz** [comment]
@RaneWallin let me know what you think. i added an authActions, eventActions, and types file

**Comment by bcuz** [comment]
[login component connected to redux](https://trello.com/c/FdFLHex8/63-login-component-connected-to-redux)

**Pull request merged**